jovian/jovian_nb_ext/main.js

```diff
@@ -707,6 +928,49 @@ define([
         .modal("show");
     };
 
+    // disables the extension
+    function removeExtension() {
+      Jupyter.notebook.save_checkpoint();
+      remove_ext =
+        "import os\n" +
+        "os.system('jupyter nbextension disable jovian_nb_ext/main sys-prefix')\n";
```

Comment:
Its `--sys-prefix` this might be the reason why disable is not working. Anyways you can use our command itself `os.system('jovian disable-extension')` which contains the command that you have used.

Reply:
I tried both ways of fixing the problem and both give me the same result.
![disableerror](https://user-images.githubusercontent.com/37026619/78813355-bdc88a80-7981-11ea-98de-1b793b19d0d5.png)
Now it says that the extension is disabled but the jovian toolbar button is still there and working after reloading the page.

Reply:
Having a alert would interfere with the `location.reload()` manually refresh would solve this.
To keep the expected behavior i have added a `confirm` modal instead and then calling reload .